fpmsyncd/fpmlink.cpp

```diff
@@ -5,10 +5,103 @@
 #include "netmsg.h"
 #include "netdispatcher.h"
 #include "fpmsyncd/fpmlink.h"
+#include "fpmsyncd/routesync.h"
 
 using namespace swss;
 using namespace std;
 
+extern RouteSync *g_routesync;
```

Comment:
>g_routesync [](start = 18, length = 11)

You may add it as FpmLink constructor parameter, so removing this global extern variable.
Please also check name convention with nearby code.